share/spice/sports/nfl/games/sports_nfl_games.js

```diff
@@ -0,0 +1,97 @@
+(function (env) {
+    "use strict";
+    
+    var Games,
+        OPS = {
+            _checkClock: true,
+            _fixCount: true,
+            _setOT: true,
+            sequenceType: "quarter",
+            currentName: "quarter",
+            counter: "quarter",
+            name: "scoring",
+            half: 2,
+            min: 4
+        };
+    
+    env.ddg_spice_sports_nfl_games = function(apiResult) {
+
+        if (!apiResult || !apiResult.data || !apiResult.data.games || !apiResult.data.games.length) {
+            return Spice.failed('nfl_games');
+        }
+        
+        DDG.require('sports', function(){
+            Games = env.ddg_spice_sports_games;
+            
+            Games.init(NFLInit);
+        });
+        
+        var NFLInit = function () {
+                Spice.add({
+                    id: 'nfl_games',
+                    name: 'NFL Games',
```

Comment:
@sdougbrown What do you think about renaming the tabs to just `NFL`? @nilnilnil suggested.